CONTRIBUTING.md

```diff
@@ -15,35 +24,42 @@ Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to
 
 When proposing major new features or changes that may introduce an API or schema change, please make sure to communicate with the [community](http://geoblacklight.org/connect) so the full implications are understood. Likely there are ways to introduce these changes in a backwards compatible way that others may be able to help with.
 
 #### Pull request overview
+1. If there isn't already an associated issue in the repository, create one.
 1. Fork it ( http://github.com/my-github-username/geoblacklight/fork )
-2. Create your feature branch (`git checkout -b my-new-feature`)
-3. Commit your changes (`git commit -am 'Add some feature'`)
-4. Push to the branch (`git push origin my-new-feature`)
-5. Create new Pull Request
+1. Create your feature branch (`git checkout -b my-new-feature`)
+1. Commit your changes (`git commit -am 'Add some feature'`)
+1. Push to the branch (`git push origin my-new-feature`)
+1. Create new Pull Request
+1. Mention the pull request in the associated issue.
 
 #### Merging Changes
 
-- It is considered "poor form" to merge your own request.
 - Please take the time to review the changes and get a sense of what is being changed. Things to consider:
-  - Does the commit message explain what is going on?
-  - Does the code changes have tests? _Not all changes need new tests, some changes are refactorings_
+  - Do the commit messages explain what is going on?
+  - Do the code changes have tests? _Not all changes need new tests, some changes are refactorings_
   - Do all new methods, modules, and classes have comments? Do changed methods, modules, and classes have comments?
   - Does the commit contain more than it should? Are two separate concerns being addressed in one commit?
-  - Did the Travis tests complete successfully?
-- If you are uncertain, bring other contributors into the conversation by creating a comment that includes their @username.
-- If you like the pull request, but want others to chime in, create a +1 comment and tag a user.
+  - Did the GitHub Actions CI tests complete successfully?
+- It is considered "poor form" to merge your own request.
+  - Given the multi-institutional nature of the project, communitiy code review is important.
+  - Bring the [GeoBlacklight Developers Team](https://github.com/orgs/geoblacklight/teams/geoblacklight-developers) into the conversation by creating a comment that tags their @username: `@geoblacklight/geoblacklight-developers`.
+  - If you want specific others to chime in, create a +1 comment and tag them.
 
-If you wish to ask questions or participate further, email the [GeoBlacklight Working Group](https://groups.google.com/forum/#!forum/geoblacklight-working-group) at [geoblacklight-working-group@googlegroups.com](mailto:geoblacklight-working-group@googlegroups.com).
+### GeoBlacklight Software Versioning
```

Comment:
Is the contribution guide the best place for this information? 

I wonder if it might be better suited for [Software Releases](https://geoblacklight.org/docs/releases/), or another page in the documentation. It seems out of place to me because most contributor will not be cutting new versions.

Reply:
Yeah I think that's why it sort of ended up on the bottom. It feels important, but is it?

Reply:
Thoughts on this, @karenmajewicz or @ewlarson?

Reply:
We removed this section as it was more about releases than contributions.